src/sidebar/components/Panel.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef PanelProps
+ * @prop {string} title
+ * @prop {import("preact").ComponentChildren} children
+ */
+
+/**
+ * Render a simple, consistent "panel"-like interface
+ *
+ * @param {PanelProps} props
+ */
+export default function Panel({ title, children }) {
+  return (
+    <div className="Panel">
+      <div className="Panel__header">
+        <div className="Panel__title">{title}</div>
```

Comment:
The corresponding element in `SidebarPanel` uses a semantic `h2` for the heading. I could imagine that in future we might make other small visual or a11y changes to `SidebarPanel` and forget to incorporate them in `Panel`.

Given that the visual structure of the `Panel` is the same as `SidebarPanel`, except for the lack of a close button, perhaps it would make sense to refactor this so that:

- Panel is a simple/pure component that renders the panel UI with a set of props
- SidebarPanel wraps panel to add the sidebar-specific integration

In other words, this would be conceptually similar to say `Menu` vs `UserMenu`.

Thoughts?